src/containers/Header/Header.tsx

```diff
@@ -80,15 +81,22 @@ function Header({mainPage}: HeaderProps) {
                         items={breadcrumbItems}
                         lastDisplayedItemsCount={1}
                         firstDisplayedItemsCount={1}
-                        renderItemContent={({icon, text}) => {
-                            if (!icon) {
-                                return text;
-                            }
-                            return (
+                        renderItem={({item, isCurrent}) => {
+                            const {icon, text, link} = item;
+
+                            const content = icon ? (
                                 <span className={b('breadcrumb')}>
                                     <div className={b('breadcrumb__icon')}>{icon}</div>
                                     {text}
                                 </span>
+                            ) : (
+                                text
+                            );
+
+                            return (
+                                <InternalLink to={isCurrent ? undefined : link}>
+                                    {content}
+                                </InternalLink>
```

Comment:
Can we keep old styles, but with new behaviour?

Styles should be like this:
<img width="365" alt="Screenshot 2024-06-13 at 16 59 35" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/d7347272-cffb-4d26-9c71-9a176a41574e">

But not they are:
<img width="369" alt="Screenshot 2024-06-13 at 16 59 40" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/48907dd7-1912-44d3-aaf6-b335325f77b4">